packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -244,8 +244,25 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       props.onKeyDown(ev);
     }
 
+    // Handle copy if focus is in the selected items list
```

Comment:
i would just add a comment here mentioning that this is a workaround we are doing till we figure out the right way of doing this, or maybe a WI number, basically something to know that we need to update this in future :)

Reply:
Added a comment in the newest iteration